data-reconciliation-app/Makefile

```diff
@@ -88,4 +88,14 @@ deploy-mccf: ## 🚀 Deploy Managed CCF
 	@cd .. && $(MAKE) deploy-mccf
 
 deploy-ms-idp: ## 🔐 Create an Identity Provider
-	@cd .. && $(MAKE) deploy-ms-idp
\ No newline at end of file
+	@cd .. && $(MAKE) deploy-ms-idp
+
+publish-app-sandbox: build ## 🧪 Deploy the Data Reconciliation Application to a network
+	@echo -e "\e[34m$@\e[0m" || true
+	@. ../scripts/submit_proposal.sh --network-url "https://127.0.0.1:8000" --proposal-file ./dist/set_js_app.json --certificate_dir "./workspace/sandbox_common" --member-count 2
+	@. ../scripts/submit_proposal.sh --network-url "https://127.0.0.1:8000" --proposal-file ./workspace/proposals/set_jwt_issuer_ms_proposal.json --certificate_dir "./workspace/sandbox_common" --member-count 2
+
+publish-app-mccf: build ## 🧪 Deploy the Data Reconciliation Application to a network
```

Comment:
I wonder if this command is better when we call it `deploy-app-to-mccf`?

Reply:
Done